experiments/AMIP/modular/components/land/bucket_init.jl

```diff
@@ -188,27 +189,35 @@ function bucket_init(
     α_snow = FT(0.8) # snow albedo
     if albedo_type == "map_static" # Read in albedo from static data file (default type)
         # By default, this uses a file containing bareground albedo without a time component. Snow albedo is specified separately.
+        if ClimaComms.iamroot(comms_ctx)
+            albedo = BulkAlbedoStatic{FT}(regrid_dirpath, α_snow = α_snow, comms = comms_ctx)
+        end
+        ClimaComms.barrier(comms_ctx)
```

Comment:
`ClimaComms.barrier` here is the sync across all procs right? `albedo` is assigned again in the following line

Reply:
This is a temporary solution before #470 is fixed in ClimaLSM. The problem is that the albedo file download often hits race conditions if it's being done by > 1 process. The first call does the download using root only and the second call enables all processes to read the file. It's only called during initialization, so performance isn't really hurt by this.

Reply:
Ooh and `ClimaComms.barrier` just makes the processes wait until they all reach this point. We could use `broadcast` here instead of the two calls, but it doesn't really matter, since it's only a temporary solution.

Reply:
OK! This seems fine; perhaps we start a separate issue flagging this for a future PR. Resolving this comment for now.

Reply:
Sounds good. Flagged this in #470